### DISCUSSION TITLE
What should I do about spam issues or pull requests?

### DISCUSSION THREAD

**Discussion by YousufSSyed** [community/community#53233] [Pull Requests]
### Select Topic Area

General

### Body

I got a pull request for one of my projects here: https://github.com/YousufSSyed/Raindrop.io-Scripts/pull/1, which only contains one new file with a blank line, nothing else. This is spam. I just closed it but what am I supposed to do when I get spam like that?

**Comment by bbkx226** [comment]
If you receive a pull request that you suspect is spam, the best thing to do is to review the request carefully before taking any action. If the request contains only one or a few new files, and those files don't seem relevant to your project, then it's likely spam. You can close the request and leave a comment explaining why you're rejecting it.

It's also a good idea to report the spam to GitHub support, so they can investigate and take appropriate action if necessary. To do this, you can navigate to the pull request page and click on the "..." button, then select "Report abuse" from the dropdown menu. This will send a report to GitHub support, who will review the request and take appropriate action if necessary.

**Reply by YousufSSyed** [reply]
@bbkx226 If I press the three dots on the first post written by the user, click "report content", then _"I want to report abusive content or behavior"_ and finally _"I want to report SPAM, a user that is disrupting me or my organization's experience on GitHub, or a user who is using my personal information without my permission"_, it shows me three options but I don't think either is a big deal enough for me to click it / they aren't applicable enough. What should I do?

**Reply by bbkx226** [reply]
If that's the case, maybe you can try to reach out to the team by this link https://support.github.com/request and "open a support ticket", explaining the issue you are experiencing and the actions you would like GitHub to take.

**Reply by YousufSSyed** [reply]
There's no way to just write a ticket to GitHub on that page. Its just a bunch of different categories and instructions on what to do. So I just went ahead and reported the PR.

**Reply by msgilligan** [reply]
The last few days I've been receiving what appear to be bot-generated SPAM **PRs** from a series of _different_ accounts. I know they are related because they are all named `<birdname><number>` (e.g. Canary110).

The current GitHub tools are not well-suited for dealing with this and the SPAM PRs are coming in fast enough that GitHub is _rate-limiting_ my reports of SPAM PRs.

**Note:** If anyone from **GitHub Support** wants to follow up, please reference Ticket ID: 2223912
